### ISSUE TITLE
Parsing plans causes codequality to produce errors

### ISSUE THREAD

**Issue by tuxmea** [declarativesystems/onceover-codequality#21]
Manifests can be checked using `puppet parser validate`.
Plans must be checked using `puppet parser validate --tasks`

**Comment by GeoffWilliams** [comment]
Hi @tuxmea Just saw this - happy to review a PR otherwise this project is looking for a maintainer. - thanks

**Comment by tuxmea** [comment]
@bastelfreak migrate to vox?

**Comment by bastelfreak** [comment]
@dylanratcliffe since you maintain onceover, would you be interested in adopting onceover-codequality? In parallel I asked in the Vox Pupuli community if someone is using onceover and interesting in taking over maintenance as well.

**Comment by bastelfreak** [comment]
Hi @GeoffWilliams, vox pupuli is interested to adopt the project in case @dylanratcliffe declines. We probably get feedback next week, I will come back to you.

**Comment by GeoffWilliams** [comment]
That would be great! appreciate the feedbacks :)

**Comment by tuxmea** [comment]
Any update on migrating the repo?

**Comment by GeoffWilliams** [comment]
nope :(

**Comment by tuxmea** [comment]
@bastelfreak any news on migrating this to vox?

**Comment by bastelfreak** [comment]
What exactly is needed here? onceover got migrated to Vox Pupuli.

**Comment by dylanratcliffe** [comment]
Yeah I can probably put some time into maintaining this, especially if it has good tests and they are somewhat healthy. If it's moved to Vox we could just mirror the testing and release actions that we have for Onceover and it wouldn't be too hard

**Comment by rwaffen** [comment]
@dylanratcliffe hi, can we move on with the plan, to also move this gem over to voxpupuli?

**Comment by tuxmea** [comment]
@GeoffWilliams are you one of the owners of this repo? Are you OK to migrate to voxpupuli?

**Comment by tuxmea** [comment]
@dylanratcliffe @GeoffWilliams as we are now starting to use openvox in favor of puppet ruby gem: any chance to get this repo transferred to voxpupuli?

**Comment by bastelfreak** [comment]
@GeoffWilliams I send you an invitation to the voxpupuli github org. it's valid for 7 days. when you accept it, you can transfer the repository afterwards. Then please add `pmc@voxpupuli.org` as gem owner.

**Comment by GeoffWilliams** [comment]
@bastelfreak repo is migrated and I added `pmc@voxpupuli.org` as gem owner
